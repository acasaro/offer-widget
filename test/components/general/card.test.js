import React from 'react';
import { shallow } from 'enzyme';
import Card from '../../../src/components/general/card/card';

describe('Card Component', () => {
	it('should render a card component', () => {
		const card = shallow(<Card />);
		expect(card).toMatchSnapshot();
	});
});
