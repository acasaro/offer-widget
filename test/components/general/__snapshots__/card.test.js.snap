// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card Component should render a card component 1`] = `
<div
  className="card "
/>
`;
